libnavigation-base/src/main/java/com/mapbox/navigation/base/options/NavigationOptions.kt

```diff
@@ -8,6 +8,15 @@ import com.mapbox.navigation.base.typedef.TimeFormatType
 
 const val DEFAULT_NAVIGATOR_POLLING_DELAY = 1500L
 
+/**
+ * Defines navigation options
+ *
+ * @param roundingIncrement defines threshold in meters for determination is user off the route
```

Comment:
Yeah, it seems it can be removed from here as a `DistanceFormatter` can be passed in over `NavigationOptions` and `MapboxDistanceFormatter` requires `roundingIncrement`.

One confusing thing that I noticed though is that `DistanceFormatter` in `NavigationOptions` is nullable but then `MapboxTripNotification` checks if that is the case, throwing an `IllegalArgumentException` if not https://github.com/mapbox/mapbox-navigation-android/blob/5315386bca169ecb144b9862de33a22bdb6a23a5/libtrip-notification/src/main/java/com/mapbox/navigation/trip/notification/MapboxTripNotification.kt#L65-L67 Could that be problematic?